sql-statements/sql-statement-select.md

```diff
@@ -227,11 +231,10 @@ In this example, you can find the query result in `/tmp/tmp_file3` as follows:
 '3','c','3.30'<<<
 ```
 
-Now `SELECT ... INTO @variable`, `SELECT ... INTO DUMPFILE` or any [external storage](https://docs.pingcap.com/tidb/stable/backup-and-restore-storages) like S3 or GCS is not supported.
-
 ## MySQL compatibility
 
 - The syntax `SELECT ... INTO @variable` is not supported.
+- The syntax `SELECT ... INTO DUMPFILE` is not supported.
 - The syntax `SELECT .. GROUP BY expr` does not imply `GROUP BY expr ORDER BY expr` as it does in MySQL 5.7. TiDB instead matches the behavior of MySQL 8.0 and does not imply a default order.
 - The syntax `SELECT ... TABLESAMPLE ...` is a TiDB extension and not supported by MySQL.
```

Comment:
We *could* mention https://bugs.mysql.com/bug.php?id=92759 here, but I'm not sure if we should. We may also add information here about this being compatible with other database systems and [ISO/IEC 9075-2](https://standards.iso.org/iso-iec/9075/-2/ed-6/en/)

Reply:
Looks like this has been part of the standard since ISO/IEC 9075-2:2003

Reply:
Because this section focuses on the difference between TiDB and MySQL, there seems no need to mention why MySQL does not support `SELECT ... TABLESAMPLE ...`.  Can we update the sentence as follows?

```suggestion
- The syntax `SELECT ... TABLESAMPLE ...` is a TiDB extension designed for compatibility with other database systems and the [ISO/IEC 9075-2](https://standards.iso.org/iso-iec/9075/-2/ed-6/en/) standard, but currently it is not supported by MySQL.
```